docs/Makefile

```diff
@@ -0,0 +1,20 @@
+# Minimal makefile for Sphinx documentation
```

Comment:
Do we still need these two Makefiles?

Reply:
They're still useful to be able to run `make build` as opposed to `sphinx-build -d docs/build/doctrees docs/source docs/build/html -b html -W --color`. We might use it in the future for other builders too like `doctest`, `linkcheck`, `coverage`.

Reply:
I just think that this might be difficult to maintain, and try to avoid as much as possible (two files `Makefile` and `make.bat` for the same purpose). Could you have `sphinx-build -d docs/build/doctrees docs/source docs/build/html -b html -W --color` in `tox.ini` so we can avoid this file altogether? Likely we should be able to do the others within `tox.ini` rather than creating another Makefile for this purpose.

Reply:
ok no problem I've deleted them